OligoDesigner.R

```diff
@@ -0,0 +1,80 @@
+#Assign a 15 nucleotide target sequence in lowercase atcg format, using spaces between nucleotides
+target <- "c g g c g a c g t g a a c g g"
+
+#convert the target to vector and check for validity
+check_target <- function(target) {
+  target_v <- unlist(strsplit(target, split = " "))
```

Comment:
I like the use of unlist(strsplit()) here. This is an efficient method that I can see myself using in the future.